app/assets/javascripts/start-modules.js

```diff
@@ -1,9 +1,9 @@
-//= require govuk/modules
-//= require modules/global-bar
-//= require modules/sticky-element-container
-//= require modules/toggle
-//= require modules/track-click
+// = require govuk/modules
```

Comment:
Presumably this doesn't break the way Rails asset pipeline includes these files?